client/src/components/Filter/components/Table/Table.tsx

```diff
@@ -0,0 +1,63 @@
+import style from './style.module.scss';
+import { FC } from 'react';
+
+export const Table: FC = () => {
+  return (
+    <div className="container">
```

Comment:
tables are not used nowadays

if grid or flex box can do the job, use them

last resort are tables